client/src/styles/Typers.css

```diff
@@ -0,0 +1,17 @@
+.alert-enter {
+  opacity: 0;
+  transform: scale(0.9);
+}
+.alert-enter-active {
+  opacity: 1;
+  transform: translateX(0);
+  transition: opacity 300ms, transform 300ms;
+}
+.alert-exit {
+  opacity: 1;
+}
+.alert-exit-active {
+  opacity: 0;
+  transform: scale(0.9);
+  transition: opacity 300ms, transform 300ms;
+}
```

Comment:
We would like to avoid, as much as possible, having .`css` files in our project. Could you perform these transactions using _tailwindcss_ only?